plan/point_get_plan.go

```diff
@@ -139,14 +139,19 @@ func tryFastPlan(ctx sessionctx.Context, node ast.Node) Plan {
 // tryPointGetPlan determine if the SelectStmt can use a PointGetPlan.
 // Returns nil if not applicable.
 // To use the PointGetPlan the following rules must be satisfied:
-// 1. No group-by, having, order by, limit clause.
+// 1. No group-by, having, order by clause. For limit clause, the count should at least 1 and the offset is 0.
```

Comment:
Actually, we can have group-by and order-by clause.

Reply:
ah.. yes.